README.md

```diff
@@ -1,264 +1,135 @@
+1. [Accessing Concourse](#accessing-concourse)
+1. [Finding configuration](#finding-configuration)
 
+## What does `paas-cf` contain?
+`paas-cf` separates the responsibility for configuring, deploying, running, and monitoring Cloud Foundry, from those responsibilities held by [`paas-bootstrap`](https://github.com/alphagov/paas-bootstrap).
+
+This repository does not itself contain the code that runs in an environment (for the most part), but instead serves to compose the different pieces into a cohesive whole. As a result, it contains a variety of pieces that tell only part of the story. The table under the heading [Finding configuration](#Finding-configuration) outlines some key directories and their purposes.
+
+## Deploying a new environment
+At a very high level, the [`create-cloudfoundry` Concourse pipeline](https://github.com/alphagov/paas-cf/blob/main/concourse/pipelines/create-cloudfoundry.yml) generates a [Bosh manifest](https://bosh.io/docs/manifest-v2/) which describes the virtual machines and their networking which make up the Cloud Foundry deployment, as well as the software which runs on each machine. The manifest is then [submitted](https://github.com/alphagov/paas-cf/blob/main/concourse/pipelines/create-cloudfoundry.yml#L2899) to the Bosh director configured in `paas-bootstrap`.
+
+### Pre-requisites
+Before you can get a Cloud Foundry deployment up and running, you will need the following available
+
++ [ ] A running [`deployer-concourse` instance from `paas-bootstrap`](https://github.com/alphagov/paas-bootstrap)
++ [ ] Make
++ [ ] Ruby >= 2.7
++ [ ] [GDS CLI](https://github.com/alphagov/gds-cli)
++ [ ] Access to `paas-credentials` (private repository) and tools installed
++ [ ] Connection to GDS VPN
++ [ ] Permission to assume the `Admin` role of the relevant AWS account (dev, ci, staging, production)
++ [ ] `AWS_DEFAULT_REGION` environment set the desired region for the environment
++ [ ] [cf CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) >=7
+
+### Deploy Cloud Foundry
+
+These instructions contain placeholders where the exact command may vary. The below table explains the purpose of those placeholders
+
+| Placeholder   | Purpose                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------|
+| `$ACCOUNT`    | The AWS account being targeted (for example, `dev`, `staging`)|
+| `$ENV` | The name of the environment being targeted. In the case of short lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars)|
+
+1. Log in to [Credhub](https://docs.cloudfoundry.org/credhub/) in the environment by running this and following the instructions on screen. This will take you in to a new shell session.
```

Comment:
```suggestion
1. Log in to [CredHub](https://docs.cloudfoundry.org/credhub/) in the environment by running this and following the instructions on screen. This will take you into a new shell session.
```